CONTRIBUTING.md

```diff
@@ -1,6 +1,55 @@
+- PRs should not be merged if they have had substantial changes made within two days of the meeting. Rebases, typo fixes, etc. do not count as substantial. Note, administrivia PRs that do not materially modify output documents may be processed by admins as needed.
+- Resolving PRs ("merging" or "closing with no action") will be done as a result of a motion made during a meeting, and approved.
+- Reopening a PR can be done if new information is made available, and a motion to do so is approved.
+- Any motion that does not have "unanimous consent" will result in a formal vote. See Voting.
+
+
+## Participation
+There are three categories of project participation:
+
+1 - Working Group Member. This is anyone who officially participates in the group's activities in any of our communication channels (email, github issues/PRs, meetings, etc.). Working group members can file PRs against documents and can participate in the PR voting procedure. Working Group members need to be employed by a W3C member organisation or participate as invited experts. 
+2 - Non Working Group Member. This is anyone who  participates in the group's activities in any of our communication channels (email, github issues/PRs, meetings, etc.). In contrast to Working Group members they cannot file a PR or vote on a PR. They can, however, create issues for specifcations. 
+3 - Chairs and Co-Chairs are members of the group but have the ability to perform administrative actions on behalf of the group. For example, manage the website, github repos and moderate the meetings. Their actions should be done with the knowledge and consent of the group. They also have the ability to merge/close PRs, but only per the group's approval. 
+
+## PRs
+Typically, PRs are expected to meet the following criteria prior to being merged:
+
+- The author of the PR indicates that it is ready for review by asking for it to be discussed in an upcoming meeting - eg. by adding it to the agenda document.
+- All comments have been addressed.
+- PRs that have objections/concerns will be discussed off-line by interested parties. A resolution, updated PR, will be expected from those talks.
+
+## Voting
+If a vote is taken during a meeting, the follow rules will be followed:
+
+- There is only 1 vote per participating company, or nonaffiliated individual.
+- Each participating company can assign a primary and secondary representative.
+A participating company attains voting rights by having any of the entity's assigned representative(s), in aggregate, attend 3 out of the last 4 meetings. The people listed as "primary" or "alternate" for an entity can be changed no more than once a month and can be changed by notifying one of the admins. The entity obtains voting rights after the 3rd meeting, not during.
+- Only members with voting rights will be allowed to vote.
+- A vote passes if more than 50% of the votes cast approve the motion.
```

Comment:
is 50% a typical practice? 70% sounds better for me in terms of guaranteeing better adoption and faireness

Reply:
We should follow whatever the rest of the W3C does

Reply:
W3C Groups rarely conducts formal votes nowadays, so there is no typical number. At the minimum, I suggest adding a quorum requirement before conducting a vote.